applications/DG-Max/Algorithms/DivDGMaxEigenvalue.cpp

```diff
@@ -359,9 +359,8 @@ void DivDGMaxEigenvalue<DIM>::solve(
     PetscErrorCode error;
     DGMaxLogger(INFO, "Starting assembly");
     discretization.initializeBasisFunctions(mesh_, order_);
-    discretization.computeElementIntegrands(mesh_, false, nullptr, nullptr,
-                                            nullptr);
-    discretization.computeFaceIntegrals(mesh_, nullptr, stab_);
+    discretization.computeElementIntegrands(mesh_, false, {});
```

Comment:
I am still a fan of naming bool function arguments, but it is up to your discretion

Reply:
Even better, it was always false. So I just removed it.